lib/home_page_ui.dart/cards_area.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+import 'package:smooth_page_indicator/smooth_page_indicator.dart';
+import 'package:wallet_app_ui/home_page_ui.dart/card_balance.dart';
+
+class CardsArea extends StatefulWidget {
+  const CardsArea({super.key});
+
+  @override
+  State<CardsArea> createState() => _CardsAreaState();
+}
+
+class _CardsAreaState extends State<CardsArea> {
+  final _controller = PageController();
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        SizedBox(
+          height: 200,
+          child: PageView(
+            scrollDirection: Axis.horizontal,
+            controller: _controller,
+            children: [
+              CardBalance(
```

Comment:
create a method to return these 3 items

it will have this signature

List< CardBalance > makeCardBalanceList() {
 // your code here